gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -209,6 +210,10 @@ def _validate_args(options, parsed_args):
 
 def run(argv=None):
   """Runs VCF to BigQuery pipeline."""
+  if argv:
```

Comment:
nit: I think you can combine this as `logging.info('Command ...', ' '.join(argv or sys.argv))`

Reply:
Done.